extra/pcmk_alarm_sample.sh

```diff
@@ -21,9 +21,14 @@ if [ -z $CRM_notify_version ]; then
     exit 0
 fi
 
+tstamp=`printf "%04d. " "$CRM_notify_node_sequence"`
+if [ ! -z $CRM_notify_timestamp ]; then
+    tstamp="${tstamp} $CRM_notify_timestamp (`date "+%H:%M:%S.%06N"`): " 
+fi
```

Comment:
I can see us adding a variety of sample scripts in the future. Perhaps we should create an extra/alerts directory and move this and pcmk_snmp_helper.sh to it.